src/Sorting.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace CultuurNet\UDB3\EventExport;
+namespace CultuurNet\UDB3;
```

Comment:
I think a good location for this class is now inside the `CultuurNet\UDB3\Search` namespace.